### ISSUE TITLE
Travis errors on Python3.6 PYTHON_PLUS=True

### ISSUE THREAD

**Issue by jGaboardi** [pysal/libpysal#127]
Since 12/5 [the Python 3.6 PYTHON_PLUS=True build](https://travis-ci.org/pysal/libpysal/jobs/463495648) on Travis has been failing with the following error message:

```
The job exceeded the maximum log length, and has been terminated.
```

Although it seems that no defaults had been changed between this build and the previous build, the Travis documentation recommends [changing the maximum log length](https://docs.travis-ci.com/user/enterprise/worker-configuration/#setting-maximum-log-length).

It is unclear at this point if this is a Travis issue that will clear up on its own or something that we have more control over.

**Comment by jGaboardi** [comment]
related to pysal/spaghetti#249?

**Comment by jGaboardi** [comment]
addressed in #140
